technology-overview.md

```diff
@@ -0,0 +1,78 @@
+# Technology overview
+
+[Agoric](https://agoric.com/about/) aims at enabling safe cooperation among otherwise mutually suspicious parties
+
+The way it does it is by making easy to write secure smart contracts and making it possible to run these contracts on one or several blockchains. Agoric strategy to writing smart contract is to write them in JavaScript using a framework called ERTP (Electronic Rights Transfer Protocol)
+
+
+## High-level view
+
+### JavaScript
+
+Agoric works to make [ECMAScript](https://www.ecma-international.org/publications/standards/Ecma-262.htm) (standard defining JavaScript) a suitable language for easily expressing secure smart contracts. Agoric works with ECMA TC39 (which includes browser makers and Node.js representatives) in that direction
+
```

Comment:
We can also mention that we're working with TC53 (IOT) on standardizing on SES as the extension language for IOT devices.

Reply:
I don't understand what you mean by "**extension** language"

My first guess would be that it means that TC53 will standardize using something like a `.ses` suffix, but i'm really not sure

Reply:
We should mention that tc53 is standardizing on SES as *the* JavaScript for embedded devices. But "extension" is the wrong word. "Extension" suggests something larger. SES is mostly defined by omission, and should altogether be smaller. But we're avoiding the term "subset" because it suggests to most people a syntactic or advisory subset, like Jessie. Instead, we now say that SES is a JavaScript runtime, or "the SES runtime for JavaScript".

Attn @katelynsills

Reply:
I like "SES is a secure JavaScript runtime"

Reply:
I added a note about Agoric's work with TC53 in a follow-up commit